src/main/java/org/cryptomator/windows/autostart/WinShortcutCreation.java

```diff
@@ -0,0 +1,25 @@
+package org.cryptomator.windows.autostart;
+
+import org.cryptomator.windows.common.NativeLibLoader;
+
+/**
+ * Creates Windows shortcuts (a special type of Windows shell links).
+ * <p>
+ * For more info, see https://docs.microsoft.com/de-de/windows/win32/shell/links.
+ */
+public class WinShortcutCreation {
```

Comment:
I'd prefer a class name closer to the native API, i.e. `WinShellLinks`.

Reply:
resolved in 9cd216b